tables/Dysco/threadeddyscocolumn.h

```diff
@@ -50,7 +50,7 @@ class ThreadedDyscoColumn : public DyscoStManColumn {
   virtual void setShapeColumn(const casacore::IPosition &shape) override;
 
   /** Get the dimensions of the values in a particular row.
-   * @param rownr The row to get the shape for. */
+   * The rownr parameter is not used as the shape is the same for all rows. */
```

Comment:
If I were a doxygen purist, I'd argue that the `@param` markup should be kept here, something like
```
* @param rownr This parameter is ignored as the shape is the same for all rows.
```
But feel free to ignore this.